src/api/factories/ProposalFactory.ts

```diff
@@ -99,8 +97,7 @@ export class ProposalFactory {
 
         const proposalCreateRequest = {
             submitter: proposalMessage.submitter,
-            blockStart: proposalMessage.blockStart,
-            blockEnd: proposalMessage.blockEnd,
+            daysRetention: smsgData.daysRetention,
```

Comment:
add timeStart, timeEnd, postedAt, receivedAt, expiredAt
in listingItem implementation daysRetention is expiryTime, so either rename daysRetention to expiryTime to be consistent or remove the field. It quite unnecessary field, but we could have it there to be consistent with listingitem implementation.